examples/agent_dispatch.py

```diff
@@ -22,19 +13,17 @@
 """
 
 
-async def create_explicit_dispatch(http_session: aiohttp.ClientSession):
-    agent_dispatch_service = AgentDispatchService(
-        session=http_session,
-        url=os.getenv("LIVEKIT_URL"),
-        api_key=os.getenv("LIVEKIT_API_KEY"),
-        api_secret=os.getenv("LIVEKIT_API_SECRET"),
-    )
-    dispatch_request = CreateAgentDispatchRequest(
-        agent_name=agent_name, room=room_name, metadata="my_metadata"
+async def create_explicit_dispatch():
+    lkapi = api.LiveKitAPI()
+
+    dispatch = await lkapi.agent_dispatch.create_dispatch(
+        api.CreateAgentDispatchRequest(
+            agent_name=agent_name, room=room_name, metadata="my_metadata"
+        )
     )
-    dispatch = await agent_dispatch_service.create_dispatch(dispatch_request)
     print("created dispatch", dispatch)
-    dispatches = await agent_dispatch_service.list_dispatch(room_name=room_name)
+
+    dispatches = await lkapi.agent_dispatch.list_dispatch(room_name=room_name)
     print(f"there are {len(dispatches)} dispatches in {room_name}")
```

Comment:
lg, btw we need a `await lkapi.aclose()` at the end to close out the session

Reply:
just added, thanks for the catch!